src/countOccurrences.js

```diff
@@ -18,7 +18,14 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  phrase = phrase.toLocaleLowerCase();
+  const arr = phrase.split("");
+
+  const arr1 = arr.filter(function(value) {
```

Comment:
```suggestion
  const arr1 = arr.filter((value) => {
```
Используй стрелочные функции